src/Apps/Onboarding/OnboardingApp.tsx

```diff
@@ -13,7 +13,7 @@ export const OnboardingApp: FC = () => {
   }
 
   return (
-    <OnboardingProvider onDone={handleDone}>
+    <OnboardingProvider onClose={handleDone}>
```

Comment:
small nitpicky thing- wouldnt it make sense to call this handleClose now for naming convention sake??

Reply:
Ah, its actually backwards (this is a good react idiom to memorize):
- Callback handlers defined as props are always named `onSomething` (which matches native events like `onClick`, `onMouseDown`, etc. This means something is being passed in from the _outside_
- Handlers defined within the body of a component are always `handleSomething`. This means that a function is being invoked from the _inside_.
- This makes it clear when reading code that anything with `on` is passed in externally, while anything with `handle` is managed locally. 

Here's a small example:

```tsx
const MyComponent = ({ onClose }) => {
  const handleClose = () => {
    if (onClose) {
      onClose()
    }
    
    // do something else 
  }
  
  return (
    return (
      <div onClick={handleClose}>Click me</div>
    )
  )
}
```

Reply:
Lemme know if this makes sense! can explain more

Reply:
oh ok that totally makes sense!